pkg/coredns/plugins/crosscluster/setup.go

```diff
@@ -39,36 +51,56 @@ func setup(c *caddy.Controller) error {
 }
 
 func setupController(c *caddy.Controller) (*CrossCluster, error) {
-	restConfig, err := rest.InClusterConfig()
-	if err != nil {
-		return nil, fmt.Errorf("error creating rest config: %v", err)
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
Do we want to hard code Dev Mode? Maybe this should instead be passed in as a config option.

Reply:
Fair, I think kubebuilder hardcodes it to true so we didn't give it much thought. We can make it configurable.

Reply:
Ah.  We were a bit concerned on whether that might leak creds or something.  If that's not a thing, then I think its fine to leave as-is for now.

Reply:
Yeah, it doesn't do anything special creds-wise. IIRC it just changes the logs from human-readable to json logs.